tools/releaseBuild/azureDevOps/templates/release-SDKTests.yml

```diff
@@ -35,6 +37,28 @@ jobs:
         artifact: metadata
         path: '$(Pipeline.Workspace)/releasePipeline/metadata'
 
+    - pwsh:  |
+        Import-Module "$(Build.SourcesDirectory)/build.psm1" -Force
+        New-NugetConfigFile -NugetFeedUrl $(PSInternalNugetFeed) -UserName $(PSInternalNugetFeedUserName) -ClearTextPAT $(PSInternalNugetFeedPAT) -FeedName AzDevOpsFeed -Destination '$(Build.SourcesDirectory)/test/hosting'
+
+        Get-ChildItem '$(Build.SourcesDirectory)/test/hosting'
+
+        # The above cmdlet creates a lower-case nuget.config. There also exists a NuGet.config which we needed to replace.
+        # Hence the following workaround
+
+        Move-Item -Path '$(Build.SourcesDirectory)/test/hosting/nuget.config' -Destination '$(Build.SourcesDirectory)/test/hosting/NuGet.Config' -Force -ErrorAction Continue
+        Get-ChildItem '$(Build.SourcesDirectory)/test/hosting'
+
+        if(-not (Test-Path "$(Build.SourcesDirectory)/test/hosting/NuGet.Config"))
+        {
+            throw "NuGet.Config is not created"
+        }
+        else
+        {
+          Get-Content "$(Build.SourcesDirectory)/test/hosting/NuGet.Config" -Raw
```

Comment:
please fix the indentation.

Reply:
We should be consistent in the file about using 2 or 4 spaces

Reply:
fixed.